local_groups/models.py

```diff
@@ -203,8 +218,6 @@ class Group(models.Model):
         max_length=250
     )
 
-    state_organizing_committee = models.BooleanField(default=False)
```

Comment:
Please double check this is safe. Also please be careful about A/B deploy issues. Usually before removing a field from db need to remove it from code in first deploy, then drop from db in later deploy.

Reply:
This was never used outside of admin/really at all. We're manually setting group status after deploy.

Reply:
Sounds fine to delete legacy data, but still should try to avoid downtime during deploys.